Golem/Factory.cs

```diff
@@ -6,12 +8,18 @@ namespace Golem
 {
     public class Factory : IFactory
     {
-        public Task<IGolem> Create(string modulesDir, ILoggerFactory? loggerFactory)
+        public Task<IGolem> Create(string modulesDir, ILoggerFactory? loggerFactory, bool mainnet = true)
         {
             var binaries = Path.Combine(modulesDir, "golem");
             var datadir = Path.Combine(modulesDir, "golem-data");
 
-            return Task.FromResult(new Golem(binaries, datadir, loggerFactory) as IGolem);
+            var network = Factory.Network(mainnet);
+
+            return Task.FromResult(new Golem(binaries, datadir, loggerFactory, network) as IGolem);
+        }
+
+        public static Network Network(bool mainnet) {
+            return mainnet ? Yagna.Types.Network.Polygon : Yagna.Types.Network.Holesky;
```

Comment:
Default testnet network is now `holesky`.
I had a payment time/delay related problems with `JobTests`, so I started to change things.